### PR TITLE
Fixes Invisable Animals

### PR DESCRIPTION
**Pull request by Dafoosh**
Animal models had some draw issue's.
Code may not be the best lol buts its how ive gotten it to work correctly feel free to fix anything you see wrong or inefficient.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
I don't think the if statement in necessary because it returns false regardless.

File: src/util/ped.hpp [hunk 1]
```diff
@@ -19,7 +19,19 @@ namespace big::ped
 		PLAYER::SET_PLAYER_MODEL(self::id, hash);
 		script::get_current()->yield();
 		STREAMING::SET_MODEL_AS_NO_LONGER_NEEDED(hash);
-
+		if (ENTITY::GET_ENTITY_MODEL(self::ped) != ENTITY::GET_ENTITY_MODEL(self::ped)) {
+			return false;
+		}
```
Comment (LiamD-Flop) [inline src/util/ped.hpp]: I might be confused but it looks like this will never be true.

**Review by LiamD-Flop — DISMISSED** [review]

File: src/util/ped.hpp (hunk 1 above)
Comment (Yimura) [reply src/util/ped.hpp]: Correct since both calls are made on the same tick, it's best to store model from the last tick and compare against that one.

**Comment by Yimura** [comment]
@Dafoosh please review the requested changes or this PR can't be merged into master.

**Review by Dafoosh — COMMENTED** [review]
Requested Changes Made ☺.

**Review by Yimura — APPROVED** [review]

**Pull request merged**